src/Stratis.Bitcoin.Features.Wallet/Interfaces/IWalletRepository.cs

```diff
@@ -184,10 +184,12 @@ public interface IWalletRepository
         /// <summary>
         /// Returns a history of all transactions in the wallet.
         /// </summary>
-        /// <param name="walletName">The name of the wallet to return the transactions of.</param>
-        /// <param name="accountName">An optional account name to limit the results to a particular account.</param>
+        /// <param name="account">An optional account name to limit the results to a particular account.</param>
```

Comment:
How are the results limited to a specific wallet?

Reply:
We get the wallet name from the account that is passed in's account root. This is called from:

https://github.com/stratisproject/StratisFullNode/pull/548/files#diff-481c41c671b79c78e1987b366a78efdd29721cb50dfff8bffe8881a830da91e3R847

Reply:
We won't have the wallet name if the account is passed as null here.